src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import MigrationPolicyEmptyListImage from 'images/migrationPoliciesEmptyState.png';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  Button,
+  ButtonVariant,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStatePrimary,
+  EmptyStateSecondaryActions,
+  EmptyStateVariant,
+  Title,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { migrationPoliciesPageBaseURL } from '../../utils/constants';
+
+const MigrationPoliciesEmptyState: React.FC = () => {
+  const { t } = useKubevirtTranslation();
+  const history = useHistory();
+  return (
+    <EmptyState variant={EmptyStateVariant.xs}>
+      <EmptyStateIcon icon={() => <img src={MigrationPolicyEmptyListImage} />} />
+      <Title headingLevel="h4" size="lg">
+        {t('No MigrationPolicies are defined yet')}
+      </Title>
+      <EmptyStateBody>
+        <Trans t={t} ns="plugin__kubevirt-plugin">
+          Click <b>Create MigrationPolicy</b> to create you first policy
+        </Trans>
+      </EmptyStateBody>
+      <EmptyStatePrimary>
+        <Button
+          variant={ButtonVariant.primary}
+          onClick={() => history.push(`${migrationPoliciesPageBaseURL}/form`)}
+        >
+          {t('Create MigrationPolicy')}
+        </Button>
```

Comment:
you can use a link here as button and avoid using history push

Reply:
I need the `ButtonVariant.primary` CSS styles to the button, you suggest switching to Link and adding those styles?